src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorConfigurationEndpointTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Test.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Moq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    public class RazorConfigurationEndpointTest : LanguageServerTestBase
+    {
+        public RazorConfigurationEndpointTest()
+        {
+            var services = new ServiceCollection().AddOptions();
+            Cache = services.BuildServiceProvider().GetRequiredService<IOptionsMonitorCache<RazorLSPOptions>>();
+
+            ConfigurationService = Mock.Of<IRazorConfigurationService>();
```

Comment:
Does it make sense to make this Mock Strict to ensure that for example things only get called once?

Reply:
It doesn't matter here because none of the methods in `ConfigurationService` is actually called. It is short-circuited by `TestRazorLSPOptionsMonitor`.

Reply:
I've used Strict in other places where it makes sense.